server/game/cards/11.4-MoD/GreyGhost.js

```diff
@@ -0,0 +1,63 @@
+const DrawCard = require('../../drawcard');
+
+class GreyGhost extends DrawCard {
+    constructor(owner, cardData) {
+        super(owner, cardData);
+
+        this.tracker = new ShadowsTracker(this.game, this);
+    }
+
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Choose character(s)',
```

Comment:
According to convention, titles are after the effect, so something like 'Make character(s) unable to defend', or maybe something less awkward than that. :)

Reply:
I couldn't make it any less awkward, which is why i went this way - open to suggestions